_posts/2021-02-21-goodbye-scrum.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Goodbye Scrum, Hello Agile
+date: 2021-02-21
+permalink: /post/goodbye-scrum-hello-agile
+description: A story about why (and a little bit of how) we adopted Kanban.
+layout: default
+redirect_from:
+  - /goodbye-scrum-hello-agile
+---
+
+# Goodbye Scrum, Hello Agile
+
+February 21, 2021
+
+Leading into early 2020 we were doing what we called Scrum-Lite. Although, to be pedantic, it was closer to elements of [XP](http://www.agile-process.org/) with more-familiar Scrum terminology. Our (only) ceremonies were daily Standups (dev + product + design) and weekly Sprints and Demos. We didn't carry a Backlog, so ceremonies surrouding that were unnecessary.
+
+Things were going alright, but at Drip we stress a culture of [kaizen](https://en.wikipedia.org/wiki/Kaizen) &mdash; one of continuous improvement. I was therefore compelled to ask "How can we do even better?" Valuing Lean and agility over dogmatic adherence to a specific Agile implementation, I saw an opportunity to address a specific issue we were facing &mdash; that of stalled projects &mdash; by tweaking our process.
+
+What follows is an document I used back in March of 2020 outlining the issue and proposing an solution to address it. The at the end is a recent relection on how Kanban is working for us.
+
+## Background
+
+I am encouraged by how we are progressing! Looking past the horizon, however, I don't think we've solved the fundamental problem that I've heard expressed by the team in various ways, including in our most recent retrospective: projects, at some point, have a tendency to feel like they are stalling.
+
+Additionally, I know that we enjoyed mobbing. It was invigorating and engaging, and, importantly, it helps minimize knowledge silos. Everyone seemed energized by the proposal that as our tasks wind down, we ramp up mobbing. However, mobbing has largely dropped by the wayside, and I don't think anything besides overcoming inertia has prevented us from mobbing more.
+
+One of our company's strategic priorities is "Unleash our people." And I would not be doing my job if I weren't constantly thinking about how to help you all be your best (professional) selves. Long story short, based on conversations I've had with [my peer] [Stephanie](https://www.linkedin.com/in/stephanie-simons-she-her-92444ab2/), who has a bunch of experience in using Kanban with her teams, I see it as a potential pathway to addressing these two issues.
```

Comment:
Saying "you all" here implies that you're writing this to the people you manage rather than to the audience reading it.

Reply:
The audience is people who are are process decision-makers and so possibly managing folks. I'll make the caveat saying this is an _internal_ document that I'm sharing publicly.